apps/javascript/src/portabl.js

```diff
@@ -0,0 +1,65 @@
+import { v4 as uuidv4 } from 'https://jspm.dev/uuid';
+
+const API_BASE_URL = window._env_.JS_APP_PUBLIC_API_HOST;
+const CLIENT_ID = window._env_.JS_APP_PUBLIC_PORTABL_PUBLIC_CLIENT_ID;
+const DOMAIN = window._env_.JS_APP_AUTH0_DOMAIN;
+const AUDIENCE = window._env_.JS_APP_AUTH0_API_AUDIENCE;
+const PASSPORT_URL = window._env_.JS_APP_PUBLIC_PASSPORT_URL;
+const SYNC_ACCEPT_URL = window._env_.JS_APP_PUBLIC_SYNC_ACCEPT_URL;
+const USER_CONSENT = '/user-consent';
+const SYNC_PREREQS = '/sync-prereqs';
+
+// used for playground purposes only
```

Comment:
I think we can remove this comment -- future integrators wouldn't know what playground is anyway. Instead we can specify that this MOCK_USER_ID should be a unique id they can use to identify their users with within the Portabl system.